Source/ThirdParty/TurboBadger/tb_widgets.cpp

```diff
@@ -1355,6 +1355,7 @@ void TBWidget::InvokePointerUp(int x, int y, MODIFIER_KEYS modifierkeys, bool to
 {
     int ox = x;
     int oy = y;
+    TBWidget *old_capture;
```

Comment:
really should initialize pointers, other than maybe in tiny amounts of code where you can guarantee they are set, I am starting to get suspicious with the amount of changes that are seeming to be necessary here :)